src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -29,7 +29,7 @@
                     #()))
 
 (defn- send-tokens [{:keys [web3 from to value gas gas-price symbol network]}]
-  (let [contract (:address (tokens/symbol->token (ethereum/network->chain-keyword network) symbol))]
+  (let [contract (:address (tokens/symbol->token (keyword (ethereum/network-names network)) symbol))]
```

Comment:
This was a regression from another PR? `network-names` sounds weird if it returns a single name?

Reply:
Yes, but I couldn't fix one without the other because intermediate solution would be so broken it's dangerous.

Reply:
Sounds good! Name is weird but we can address that later.